bingo/tests/dbc/BingoElastic.py

```diff
@@ -36,6 +38,8 @@ def import_data(self, data_path: str, database_type: str):
         index = 1
         for compound in indigo_iterator(self.indigo, data_path):
             try:
+                if compound.checkBadValence():
+                    raise IndigoException(compound.checkBadValence())
```

Comment:
We should leave it to user and his session settings. If he does not want to take account of the valence, then we also should not.

Reply:
I agree, but if, for example, a user wants to save all broken molecules, is it correctly to have them in the search results? And if not (our tests say - no), at what point it would be better to skip them? I decided to nip them in the bud, which is probably too harsh.

Reply:
Removed this check from here and added as a filtration parameter to Elastic query.